apps/meteor/tests/end-to-end/api/livechat/06-integrations.ts

```diff
@@ -115,6 +115,46 @@ describe('LIVECHAT - Integrations', () => {
 						expect(res).to.have.property('text', '<Response></Response>');
 					});
 			});
+
+			it("should set visitor's source as SMS after sending a message", async () => {
+				await updateSetting('SMS_Default_Omnichannel_Department', '');
+				await updateSetting('SMS_Service', 'twilio');
+
```

Comment:
Let's undo these changes `after` all...

Reply:
Also can we delete the visitor too?

Reply:
I'm now resetting both settings in the `after` statement
Regarding visitors, isn't it some sort of pattern from our side not to delete them? :thinking: I could be wrong, though

Reply:
I'm now deleting visitors here too